### ISSUE TITLE
high loss in validation when I train

### ISSUE THREAD

**Issue by yustaub** [ZQPei/deep_sort_pytorch#94]
I download the market1501 dataset and reorganize the train and test dataset, but when I train the model, the train accuracy increase but the test accuracy is alwanys very low, 
Epoch : 40
[progress:9.9%]time:5.85s Loss:0.19229 Correct:1227/1280 Acc:95.859%
[progress:19.7%]time:12.49s Loss:0.19317 Correct:2455/2560 Acc:95.898%
[progress:29.6%]time:6.21s Loss:0.16215 Correct:3698/3840 Acc:96.302%
[progress:39.4%]time:6.23s Loss:0.18000 Correct:4937/5120 Acc:96.426%
[progress:49.3%]time:6.22s Loss:0.18456 Correct:6169/6400 Acc:96.391%
[progress:59.1%]time:6.27s Loss:0.18482 Correct:7395/7680 Acc:96.289%
[progress:69.0%]time:6.28s Loss:0.19110 Correct:8616/8960 Acc:96.161%
[progress:78.8%]time:6.00s Loss:0.17664 Correct:9854/10240 Acc:96.230%
[progress:88.7%]time:6.27s Loss:0.16078 Correct:11090/11520 Acc:96.267%
[progress:98.5%]time:6.28s Loss:0.17974 Correct:12330/12800 Acc:96.328%
Testing ...
[progress:100.0%]time:41.70s Loss:13.60742 Correct:24/15913 Acc:0.151%
Learning rate adjusted to 0.0010000000000000002
 anyone can sovle this?

**Comment by montaboom** [comment]
im having the same problem im trying to train it on my own dataset of cars .. any solu!

**Comment by yustaub** [comment]
@montaboom sorry , i haven't solve it

**Comment by DerekSunYH** [comment]
你好，请问训练时除了改输入数据集路径还要改啥吗？我也下了1501的数据集，改了输入路径，但是程序报错：RuntimeError: Found 0 files in subfolders of: /Users/sunyuhua/Desktop/Market1501/bounding_box_train
Supported extensions are: .jpg,.jpeg,.png,.ppm,.bmp,.pgm,.tif,.tiff,.webp   
请大神赐教

**Comment by yustaub** [comment]
@DerekSunYH 
you should divide the Market1501 to different folders such as 0000,0001,....

**Comment by DerekSunYH** [comment]
all pictures are in file folder "bounding_box_train"。What should I divide them based on？

**Comment by montaboom** [comment]
@yustaub is there any other repo that you know where i can train the tracker to track cars ..?

**Comment by yustaub** [comment]
@montaboom sorry , i didn't know ,did you solve this problem?

**Comment by MrWhiteHomeman** [comment]
@yustaub  Did you solve this problem ?  I meet this same problem

**Comment by joaquinCaceres** [comment]
When I download the MARS dataset, I saw that train and test dataset have different classes. Solution--> The train and test dataset must have the same classes.

**Comment by shekarneo** [comment]
@joaquinCaceres  yes you are right for market 1501 dataset we need to split train dataset into train and validation. not to use test set as validation.
